Library/Homebrew/test/unpack_strategy/zstd_spec.rb

```diff
@@ -6,5 +6,7 @@
 describe UnpackStrategy::Zstd do
   let(:path) { TEST_FIXTURE_DIR/"cask/container.tar.zst" }
 
-  include_examples "UnpackStrategy::detect"
+  it "is correctly detected" do
+    expect(UnpackStrategy.detect(path)).to(be_a(described_class).or(be_a(UnpackStrategy::Tar)))
+  end
```

Comment:
`UnpackStrategy.detect(path)` for a `.tar.XXX` file returns either `UnpackStrategy::Tar` if the host's `tar` is able to extract that compressed file or `UnpackStrategy::XXX` otherwise (such as `UnpackStrategy::Zstd`). On macOS `UnpackStrategy.detect("container.tar.zst")` returns `UnpackStrategy::Zstd`, and on `ubuntu-22.04` it returns `UnpackStrategy::Tar`, because the host's version of `tar` is recent enough and `zstd` is installed.

Reply:
This seems worth documenting in a comment, if not adjusted in `shared_examples.rb`.

Reply:
Done!